stream/dub.sdl

```diff
@@ -14,19 +12,25 @@ importPaths "."
 copyFiles "../lib/win-i386/libeay32.dll" "../lib/win-i386/ssleay32.dll" platform="windows-x86"
 copyFiles "../lib/win-amd64/libeay32.dll" "../lib/win-amd64/ssleay32.dll" platform="windows-x86_64"
 
-configuration "windows-mscoff" {
+// LDC 1.0.0 doesn't include libz in libphobos
+libs "z" platform="posix-ldc"
+libs "z" platform="gdc"
+
+configuration "openssl-mscoff" {
```

Comment:
Yeah, the next step would definitely be to separate this into a vibe-d:tls package. Maybe I should even better do that now... Having an abstraction independent of vibe.d's streams also makes sense in principle. It should of course also be noted that there are two other obvious TLS implementation choices, namely the ones provided by the OS on Windows and macOS.